src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java

```diff
@@ -74,42 +73,42 @@ public static void checkUser(Request req) {
                     LOG.info("New user is creating self. OK to proceed without existing user object for auth0UserId");
                 } else {
                     // Otherwise, if no valid user is found, halt the request.
-                    logMessageAndHalt(req, HttpStatus.NOT_FOUND_404, "Unknown user.");
+                    JsonUtils.logMessageAndHalt(req, HttpStatus.NOT_FOUND_404, "User is unknown to Auth0 tenant.");
```

Comment:
I think we discussed that `logMessageAndHalt` and other "well-known" static imports are OK (i.e., we should remove the qualifer). Also, this message is incorrect. If we get to this point, the Auth0 user exists in the tenant, but we do not have a user in our MongoDB database. A better message might be:

> No user found in database associated with the provided auth token.

Reply:
Updated.